add gettext-tools as build dependency

conda-forge-admin opened the pull request:
Hi! This is the friendly automated conda-forge-webservice.

I've started rerendering the recipe as instructed in #22.

If I find any needed changes to the recipe, I'll push them to this PR shortly. Thank you for waiting!



Here's a checklist to do before merging.
- [ ] Bump the build number [if needed](https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-recipes)

Fixes #22

conda-forge-webservices[bot] committed:
dummy commit for rerendering

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.14.10.19.0

hmaarrfk committed:
Increment build number from 1 to 2

hmaarrfk committed:
Update source URL for libidn2

hmaarrfk committed:
Update SHA256 checksum in meta.yaml

hmaarrfk commented on recipe/meta.yaml:
```diff
@@ -5,12 +5,12 @@ package:
-  url: https://ftp.gnu.org/gnu/libidn/libidn2-{{ version }}.tar.gz
-  sha256: f557911bf6171621e1f72ff35f5b1825bb35b52ed45325dcdee931e5d3c0787a
+  url: https://gitlab.com/libidn/libidn2/-/archive/v{{ version }}/libidn2-v{{ version }}.tar.gz
+  sha256: 304933ffe87d7ed0288561983d3c63d128cda533ef1942ce2aa57876f30f2391
```
```suggestion
  url: https://ftp.gnu.org/gnu/libidn/libidn2-{{ version }}.tar.gz
  sha256: f557911bf6171621e1f72ff35f5b1825bb35b52ed45325dcdee931e5d3c0787a
```

hmaarrfk committed:
Apply suggestion from @hmaarrfk

hmaarrfk committed:
Try to cross compile

hmaarrfk committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.14.10.19.06

Other tools:
- conda-build 25.9.0
- rattler-build 0.47.1
- rattler-build-conda-compat 1.4.6

hmaarrfk committed:
revert back to the ftp server.

Revert source URL and SHA256 for libidn2 to GNU FTP server.
